revenue_app/templates/revenue_app/dashboard.html

```diff
@@ -8,5 +8,155 @@
 
 {% block content %}
 <h1>Dashboard</h1>
-
+<br>
+<div class="container mt-4">
+  <div class="row">
+    {% for currency, data in summarized_data.items %}
+    <div class="col-6">
+      <h4>{{currency}}</h4>
```

Comment:
In the previous commit I saw here Argentina and Brazil, is the name currency ok?

Reply:
Not necessary, but we only have the data about the currency, so that's the filter we are using. We can put ARS and BRL instead of Argentina and Brazil to make it more accurate.